doc/DOCUMENTATION.org

```diff
@@ -435,6 +436,9 @@ Example:
   (setq <layer>-excluded-packages '(package1 package2 ...)
 #+END_SRC
 
+**** Replace layer package with local version (e.g. for contributing to a package)
```

Comment:
I think you can take the parenthesis out of the section title.
So the auto-generated TOC entry won't be so long.

Reply:
Actually this can be named as
`Overriding a layer package`

Reply:
Thanks! Good feedback. I agree this would be comparably informative, while looking better